packages/ui/src/components/va-stepper/VaStepper.vue

```diff
@@ -307,5 +338,17 @@ export default defineComponent({
     display: flex;
     gap: var(--va-stepper-controls-gap);
   }
+
+  .invalid-navigation.va-stepper__step-button--active{
+    color: var(--va-danger);
+    --30407a18-stepperColor: var(--va-danger) !important;
```

Comment:
You  mustn't do that... 
We use v-bind in vue which adds a uniq hash for css variable. It means this variable will be different every build. You need to pass `danger` to Button via props.

Reply:
Can you check my last commit? I would think this would work, but the button color doesnt seem to react to the new button color. I figured that I could use a computed prop, but from what I'm seeing I cant pass a parameter to a computed prop.